app/views/root/_ga_postcode_error_tracking.html.erb

```diff
@@ -1,9 +1,11 @@
-<script type="text/javascript">
-  GOVUK.analytics.trackEvent(
-    "userAlerts:<%= alert_type %>",
-    "postcodeErrorShown:<%= @location_error.postcode_error %>",
-    {
-      label: "<%= t(@location_error.message, @location_error.message_args) %>"
-    }
-  );
-</script>
+<% if @location_error && @location_error.postcode_error %>
+  <script type="text/javascript">
```

Comment:
That is something that we should definitely change this to, but for this story have opted to keep it as it is since there are a number of different postcode errors used in both local authorities and find my nearest. We were intending on writing another story to make this change.